bin/typedoc

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env node
 
-var td = require('../dist/lib/cli.js');
+const td = require('../dist/lib/cli.js');
```

Comment:
I went ahead and put the `var` back here. It should be fine, as that usage of `const` is ok in Node 4, but just to be consistent, it seemed better to put it back.